### ISSUE TITLE
Frenzy/Power/Endurance Charges missing for minions

### ISSUE THREAD

**Issue by Sirethas** [PathOfBuildingCommunity/PathOfBuilding#1994]
So I noticed a bug while planning for Summon Skeleton build (Necromancer Witch), and unfortunately the bug is very hard to reproduce but I suspect this seem happen only after update. 
Steps I took to make this happen:
I was doing non summoner build, saved it and then created new one.
I setup gems, passive tree and all that stuff, all while not going to configuration page.
While having Vaal Summon Skeleton, there was no option to check Frenzy/Power/Endurance Charges for minions.

**The issue seems to fix itself when you create new summoner build, add minion gem to skill set and go right to the configuration**, unfortunately, the previous build is bricked because the option to check charges won't appear there but it appears to any minion build you do afterwards (even after I restarted the PoB client). I spot this issue on 2 different PCs and on 2 different versions:

**1.4.170.20** and today at **1.4.170.25**

This is just assumption, but I think this bug is reproduceable after every new version when self-fixed. I will try to reproduce it in other ways as well, will let you know if I manage to do it. Furthermore I can't tell if only summon skeletons is the issue or it's its vaal version or its minions in general. I will try to investigate a bit more

**Comment by Wires77** [comment]
Please submit a build pastebin so we can investigate the bricked build.

**Comment by Wires77** [comment]
Actually, this is a duplicate of #1691 If you change your skill on the Calcs tab to a minion skill, the charges config should show up.
